OpenRA.Game/Network/Session.cs

```diff
@@ -122,6 +117,8 @@ public IEnumerable<Client> NonBotPlayers
 
 		public enum ClientState { NotReady, Invalid, Ready, Disconnected = 1000 }
 
+		public enum ConnectionQuality { Good, Moderate, Poor }
```

Comment:
Not a fan of having these inside the class when they are also used outside this class, only makes the access a bit more cumbersome.

Reply:
I agree, but this is standard for everything in `Session` so it would be weird to *not* do that for this.

We should fix this more generally, but that would increase the scope of this PR by a factor of several.